src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final String[] colors
+        = new String[]{"blue", "yellow", "green", "grey", "pink", "orange"};
```

Comment:
```suggestion
    private static final String[] COLORS
        = new String[]{"blue", "yellow", "green", "grey", "pink", "orange"};
```
let's make it a constant